config/file_config.go

```diff
@@ -285,6 +285,7 @@ type PeerManagementConfig struct {
 
 type RedisPeerManagementConfig struct {
 	Host           string   `yaml:"Host" cmdenv:"RedisHost"`
+	ClusterHosts   []string `yaml:"ClusterHosts"`
```

Comment:
What do you think of this also being a `cmdenv` option so that cluster hosts can be provided in an environment variable?

Reply:
added the environment variable support